src/DocumentFormat.OpenXml/SimpleTypes/OpenXmlComparableReferenceType.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System;
+
+namespace DocumentFormat.OpenXml
+{
+    /// <summary>
+    /// Represents a comparable and equatable reference.
+    /// </summary>
+    /// <typeparam name="T">The type of the value.</typeparam>
+    public abstract class OpenXmlComparableReferenceType<T> : OpenXmlSimpleType,
+        IComparable, IComparable<OpenXmlComparableReferenceType<T>>, IEquatable<OpenXmlComparableReferenceType<T>>
+        where T : class, IComparable, IComparable<T>, IEquatable<T>
+    {
+        /// <summary>
+        /// Creates a new instance of <see cref="OpenXmlComparableReferenceType{T}"/>.
+        /// </summary>
+        protected OpenXmlComparableReferenceType()
```

Comment:
Can you make these constructors `private protected`? I'd like to limit visibility of new types/methods as much as possible unless we have a reason to make them public

Reply:
Sure, can do.

Reply:
What about the existing constructors of `OpenXmlSimpleType` and `OpenXmlSimpleValue<T>`, which are all `protected`? Should we consider making those `private protected` in v3.0?